qemu/tests/virtio_fs_group_permission_access.py

```diff
@@ -61,7 +62,10 @@ def run(test, params, env):
                     else:
                         test.fail("Unknown error when deleting the "
                                   "user: %s" % o)
-            process.run(add_user_cmd % _username)
+            if process.system("grep %s /etc/group" % _username, shell=True,
+                              ignore_status=True) == 0:
+                add_cmd = "useradd -g %s %s" % (_username, _username)
+            process.run(add_cmd)
```

Comment:
Hi @xiagao 
I found no steps for deleting the `usergroup` in this file, unlike the following file.

Reply:
Generally speaking, the group is supposed to be deleted when delete user with cmd "    del_user_cmd = "userdel -r -f %s"
", but sometimes the group existed while the user was not there, therefore adding L65 to L67 to adapt such situation.